bin/mops-alertsBackup

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node --harmony
+
+'use strict';
+
+var program = require('commander');
+var prompt = require('prompt');
+var MMS = require('node-mms-client');
+var q = require('q');
+
+var MopsConfig = require('../lib/util/MopsConfig');
+
+program
+  .option('-g, --groupId [groupId]', 'MMS Group ID')
+  .parse(process.argv);
+
+var groupId = program.groupId;
+var requiredFields = [];
+if (!groupId) {
+    requiredFields.push('groupId');
+}
+
+if (requiredFields.length > 0) {
+    prompt.message = "Required Field";
+    prompt.start();
+    prompt.get(requiredFields, function (err, result) {
+        if (err) {
+            console.log("An error occurred");
+        } else {
+            if (!groupId) {
+                groupId = result.groupId;
+            }
+            alertsBackup(groupId);
+        }
+    });
+} else {
+    alertsBackup(groupId);
+}
+
+function alertsBackup(groupId) {
+    var config = new MopsConfig();
+    writeAlertsConfig (groupId, config.get('host'), config.get('user'), config.get('apiKey'));
+}
+
+function writeAlertsConfig(groupId, host, user, apiKey) {
+    var mms = new MMS({
+        username: user,
+        apiKey: apiKey,
+        host: host
+    });
+    var alertsConfig = mms.groups(groupId).alertconfigs().list(printResponseCallback("Group AlertConfigs List"));
+    q.all(alertsConfig).then(
```

Comment:
Just tried it, looks like no-go on that one:
```
$ ./bin/mops-alertsExport -g 533850d5541410e80d3d0036

/Users/petercgravelle/git/mops-cli/bin/mops-alertsExport:51
    q.alertsConfig.then(
                   ^
TypeError: Cannot call method 'then' of undefined
    at writeAlertsConfig (/Users/petercgravelle/git/mops-cli/bin/mops-alertsExport:51:20)
    at alertsBackup (/Users/petercgravelle/git/mops-cli/bin/mops-alertsExport:41:5)
    at Object.<anonymous> (/Users/petercgravelle/git/mops-cli/bin/mops-alertsExport:36:5)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
```

Gonna stick with the seemingly extraneous all